cypress/e2e/circulation-log/same-date-and-time-format-as-fee-fine-details-page.cy.js

```diff
@@ -90,8 +90,8 @@ describe('circulation-log', () => {
   });
 
   it(
-    'C350712 Check date and time --fee/fines (Firebird) (TaaS)',
-    { tags: [TestTypes.extendedPath, DevTeams.firebird] },
+    'C350712 Check date and time --fee/fines (Volaris) (TaaS)',
```

Comment:
Ususally dev team in test description is written in lovercase

Reply:
fixed